modules/lead/toolchain/prometheus-values.tpl

```diff
@@ -43,6 +43,13 @@ prometheusOperator:
     enabled: false
 prometheus:
   prometheusSpec:
+    storageSpec:
+      volumeClaimTemplate:
+        spec:
+          accessModes: ["ReadWriteOnce"]
+          resources:
+            requests:
+              storage: 1Gi
```

Comment:
is 1Gi enough storage for prometheus?

Reply:
probably not... try it in sandbox and see how long it takes to run out of space?